generation/maisi/maisi_train_vae_tutorial.ipynb

```diff
@@ -30,20 +30,10 @@
    "execution_count": 1,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !python -c "import xformers" || pip install -q xformers --index-url https://download.pytorch.org/whl/cu121
</pre><p>Can also remove xformers here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done. Thanks

Reply:
Hi @Can-Zhao , similarly, can we remove the same line in `generation/maisi/maisi_inference_tutorial.ipynb`?

Reply:
Sorry. I forgot that. Done. Thanks! Also removed description on xformer installation in readme.